fast64_internal/oot/oot_level.py

```diff
@@ -169,18 +169,31 @@ def onUpdateOOTEmptyType(self, context):
 
 class OOT_ObjectProperties(bpy.types.PropertyGroup):
 	version: bpy.props.IntProperty(name="OOT_ObjectProperties Version", default=0)
-	cur_version = 0 # version after property migration
+	cur_version = 1 # version after property migration
```

Comment:
the default of `version` should be cur_version, as new objects will be on the current version obviously. and old objects already have this property set so changing the default won't change those
this also means you could then remove setting `version = cur_version` in the `OOT_AddScene` and `OOT_AddDoor` operators, and maybe elsewhere